### DISCUSSION TITLE
HttpError 429 when requesting ... message': 'Resource has been exhausted (e.g. check quota)

### DISCUSSION THREAD

**Discussion by mdebets** [GAM-team/got-your-back#460] [Q&A]
The last few days (I think since I have upgraded to 1.80 but I'm not sure),  I always get the error "HttpError 429 when requesting ... message': 'Resource has been exhausted (e.g. check quota)" when trying to back-up my gmail account. As suggested elsewhere, I didn't run the back-up over the weekend in the hope, that the quota would reset, but it didn't and the error still happens today. 
Does anyone know, what else could be done, to fix the error?

**Comment by mdebets** [comment]
Found the solution somewhere else.
I added the --batch-size 20 argument and the back-up is up and running again without the error.

**Reply by Shaav** [reply]
Thank you for that!
